pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/stats/MetadataStoreStats.java

```diff
@@ -76,12 +60,12 @@ public final class MetadataStoreStats implements AutoCloseable {
     public MetadataStoreStats(String metadataStoreName) {
         this.metadataStoreName = metadataStoreName;
 
-        this.getOpsSucceedChild = GET_OPS_SUCCEED.labels(metadataStoreName);
-        this.delOpsSucceedChild = DEL_OPS_SUCCEED.labels(metadataStoreName);
-        this.puttOpsSucceedChild = PUT_OPS_SUCCEED.labels(metadataStoreName);
-        this.getOpsFailedChild = GET_OPS_FAILED.labels(metadataStoreName);
-        this.delOpsFailedChild = DEL_OPS_FAILED.labels(metadataStoreName);
-        this.putOpsFailedChild = PUT_OPS_FAILED.labels(metadataStoreName);
+        this.getOpsSucceedChild = OPS_SUCCEED.labels(metadataStoreName, OPS_TYPE_GET);
```

Comment:
why `this.` prefix? There is no collision in constructor arguments.

Reply:
personal code style